tests/testthat/_snaps/bind.md

```diff
@@ -410,11 +410,11 @@
       # Integers as rows
       suppressMessages(with_memory_prof(vec_rbind_list(ints)))
     Output
-      [1] 2.77KB
+      [1] 2.74KB
```

Comment:
Memory snapshots go down because we removed `names_curr` and `names_next` from the arg data structs